experiments/AMIP/moist_mpi_earth/coupler_utils/unit_tester.jl

```diff
@@ -27,40 +27,40 @@ domain = Domains.SphereDomain(FT(6731e3))
 mesh = Meshes.EquiangularCubedSphere(domain, 4)
 topology = Topologies.Topology2D(mesh)
 quad = Spaces.Quadratures.GLL{5}()
-boundary_space = Spaces.SpectralElementSpace2D(topology, quad)
+boundary_space_t = Spaces.SpectralElementSpace2D(topology, quad)
 
-land_mask = ClimaCore.Fields.zeros(boundary_space)
+land_mask_t = ClimaCore.Fields.zeros(boundary_space_t)
```

Comment:
Im curious about the _t subscript here :-)

Reply:
It stands for test. 😇 I know this file only a temporary way of testing, but it was overwriting the variables from the driver, which made debugging a bit harder. :-P